packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -43,16 +41,11 @@ import {getAggregationBits, getAggregatorModulo} from "./utils";
 export class AttestationService {
   private readonly config: IBeaconConfig;
   private readonly provider: IApiClient;
-  // order is important
-  private readonly secretKeys: SecretKey[] = [];
-  // order is important
-  private readonly publicKeys: BLSPubkey[] = [];
-  // order is important
-  private readonly validators: (ValidatorResponse | null)[] = [];
+  private readonly validators: Map<string, ValidatorAndSecret> = new Map();
```

Comment:
for the sake of readability might be good to add comment what this string key really is or create string alias
`type PublicKeyHex = string; Map<PublicKeyHex, ValidatorAndSecret>`